front_end/src/app/(main)/(tournaments)/tournaments/components/tournaments_grid/tournaments_grid.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import React from "react";
+
+import { TournamentPreview } from "@/types/projects";
+import cn from "@/utils/core/cn";
+
+type Props = {
+  items: TournamentPreview[];
+  renderItem?: (item: TournamentPreview) => React.ReactNode;
+  className?: string;
+};
+
+const TournamentsGrid: React.FC<Props> = ({ items, renderItem, className }) => {
+  return (
+    <div
+      className={cn(
+        `
```

Comment:
Interesting format you chose here - any reason why? Just curious, since it's different from the other places in our code